src/referral_storage.cairo

```diff
@@ -0,0 +1,109 @@
+use starknet::{ContractAddress,ClassHash};
+use zeroable::Zeroable;
+
+
+#[starknet::interface]
+trait IReferralStorage<TContractState> {
+    fn get_referrer(
+         self: @TContractState,
+        _account: ContractAddress,
```

Comment:
Conventionaly, variables starting with "_" are for unused variables. Please change such variable names to normal at all the places.

Reply:
https://github.com/jediswaplabs/referral-contracts-cairo/commit/90ab9786fb24eb689d00a821fb5e8c0171223a55